chapter_14_simple_form.asciidoc

```diff
@@ -188,6 +188,19 @@ You might use the interactive console, or write some exploratory code
 (but you have to promise the Testing Goat that you'll throw it away
 and rewrite it properly later).
 
+Another way of discovering is to use `pdb`,
+https://docs.python.org/3/library/pdb.html[The Python Debugger].
+To use it, simply put a `breakpoint()` in your code where you
+would like it to pause and enable you to look around.footnote:[Before Python 3.7
+you would invoke the debugger by placing `import pdb; pdb.set_trace()` in your
+code.]
+Once the execution is halted, you can print variables, results of expressions,
+list the source code lines surrounding where your program is currently at,
+step your code through each line and even more! Type `h` and press enter to
+list all your options and
+https://docs.python.org/3/library/pdb.html#debugger-commands[have a look at the
+documentation].
```

Comment:
re: https://github.com/hjwp/Book-TDD-Web-Dev-Python/pull/297

Reply:
i think if we start talking about pdb, then we need a much more detailed explanation.

Reply:
so i'd rather just not mention it tbh.  i think i already put it in a footnote somewhere, i'd rather not get into it

Reply:
ACK, removing it soon